pkgs/development/python-modules/pybind11/default.nix

```
@@ -99,6 +100,10 @@ buildPythonPackage rec {
     "test_cross_module_exception_translator"
   ];
 
+  postInstallCheck = ''
```

```suggestion
  postCheck = ''
```
is `postInstallCheck` for Python packages

---

Since `doCheck` is forced to `false`, I thought this would not work, but it does, thanks !